modules/expandable-panel/ExpandablePanel.tsx

```diff
@@ -0,0 +1,229 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState
+} from "react";
+import { withStyles } from "@diana-ui/base";
+import {
+  StandardProps,
+  WithStylesProps,
+  ThemeStyleSheetFactory,
+  Theme
+} from "@diana-ui/types";
+import { useWindowSize } from "@diana-ui/hooks";
+import { Icon } from "@diana-ui/icon";
+import { TextHighlight } from "@diana-ui/typography";
+
+const SLIDE_ANIMATION_DURATION_MS = 200;
+
+// @ts-ignore
+export interface IProps extends StandardProps<"div"> {
+  disabled?: boolean;
+  expanded?: boolean;
```

Comment:
not sure why we need this prop here since the state is managed internally. am I missing something ?

Reply:
if this is for the multiple panels could we have a `useImperativeHandle` for each panel?

Reply:
`expanded` is used by `ExpandableParents` indeed, and could be used by any other parent component that sees fit to manually control the panel's state.

What is the benefit of using `useImperativeHandle` over a prop? As I understand it using it is ill-advised save for a few exceptions. (Maybe this is one of them?)

Reply:
I just suggest that so we don’t need to manage two states (here and the parent component), and it makes a little more confusing on the component itself. Nevertheless its ok for me to keep as you have :)